indico/modules/events/api.py

```diff
@@ -538,6 +512,33 @@ def _update_query(self, query):
     def serialize_events(self, events):
         return map(self._build_event_api_data, events)
 
+    def _serialize_category_path(self, category):
+        visibility = {'id': '', 'name': ''}
+        path = [self._serialize_path_entry(category_data) for category_data in category.chain]
+        if category.visibility is None:
+            visibility['name'] = 'Everywhere'
```

Comment:
I'd set id to None in this case (actually, I'd use Everywhere/None as the default in L516 and just update them in case visibility it's not None). This also removes the need of assigning everywhere twice:

```
visibility = {'id': None, 'name': 'Everywhere'}
if category.visibility is not None::
    try:
        visibility = path[-category.visibility]
    except IndexError:
        pass
```